.github/workflows/ci.yml

```diff
@@ -113,21 +76,6 @@ jobs:
         run: |
           cmake --build --preset=debug-build-${{ matrix.platform }} --target=install
 
-      - name: Build HealthGPS (release)
-        if: "!cancelled()" # Run this step, even if the previous one fails
-        run: |
-          # Build documentation so we can show Doxygen warnings
-          cmake --preset=${{ matrix.platform }}-release -DWARNINGS_AS_ERRORS=ON -DBUILD_DOC=ON
-          cmake --build --preset=release-build-${{ matrix.platform }} --target=install
-
-      - name: Upload artifacts
-        if: matrix.is_main
-        uses: actions/upload-artifact@v3
-        with:
-          name: health-gps-${{ matrix.platform }}
-          path: |
-            ${{ github.workspace }}/out/install/${{ matrix.platform }}-*/*
```

Comment:
I didn't notice this before, but we want to keep these steps! We do a release build to check it compiles and then we upload the binaries as a test artifact so to help with debugging CI failures.

The first step could be its own action, if you fancy implementing that.

Reply:
If we're including these steps in the CI workflow, do we still need them in the release workflow (seeing as the release workflow calls the CI workflow first)?

Reply:
Oh, I didn't notice that. We don't want to run the whole CI suite again for the release workflow, because `ci.yml` is already being run for every push to `main`. We just need to create release builds for the binaries (though we don't need the "upload artifacts" step here, because that's for testing only and distinct from the "upload release artifacts" step)